charts/dependency-track/values.yaml

```diff
@@ -64,8 +64,7 @@ resources:
     cpu: 1500m
     memory: 4Gi
   limits:
-    cpu: 4
```

Comment:
The limits there match dependency-track requirements (and is just a guideline). Why should it be changed?